packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -19,10 +21,17 @@ export class PluginsPage {
 	}
 
 	/**
-	 * Visit /plugins/*
+	 * Visit /plugins or /plugins/:site
 	 */
-	async visit( route?: string ): Promise< void > {
-		await this.page.goto( getCalypsoURL( `${ route || '' }` ) );
+	async visit( site = '' ): Promise< void > {
+		await this.page.goto( getCalypsoURL( `plugins/${ site }` ) );
+	}
+
+	/**
+	 * Visit /plugins/:page/ or /plugins/:page/:site
+	 */
+	async visitPage( page: string, site = '' ): Promise< void > {
+		await this.page.goto( getCalypsoURL( `plugins/${ page }/${ site }` ) );
```

Comment:
Not used yet but this will cover each of these contexts [mentioned earlier](https://github.com/Automattic/wp-calypso/pull/61224#discussion_r810688361):
```
/plugins
/plugins/:site
/plugins/:plugin
/plugins/:plugin/:site
/plugins/:category
/plugins/:category/:site
```